src/Factory/PhpredisClientFactory.php

```diff
@@ -229,24 +242,18 @@ private function createClient(RedisDsn $dsn, string $class, string $alias, array
         return $client;
     }
 
-    /**
-     * @return Redis::SERIALIZER_*
-     *
-     * @throws InvalidConfigurationException
-     */
+    /** @throws InvalidConfigurationException */
     private function loadSerializationType(string $type): int
     {
         $types = [
-            'default' => Redis::SERIALIZER_NONE,
-            'json' => Redis::SERIALIZER_JSON,
-            'none' => Redis::SERIALIZER_NONE,
-            'php' => Redis::SERIALIZER_PHP,
+            'default' => 0, // Redis::SERIALIZER_NONE,
+            'none' => 0, // Redis::SERIALIZER_NONE,
+            'php' => 1, //Redis::SERIALIZER_PHP,
+            'igbinary' => 2, //Redis::SERIALIZER_IGBINARY,
+            'msgpack' => 3, //Redis::SERIALIZER_MSGPACK,
+            'json' => 4, // Redis::SERIALIZER_JSON,
```

Comment:
Those constants are identical between with Relay and PhpRedis if you want to keep using them.

Reply:
Yeah that's why I hardcoded the values. Because I want that piece of code working no matter if Phpredis is installed or not. Easier this way rather than adding conditional to know which class to use where to get constant values from.